bindings/cpp/include/portaudiocpp/SampleDataFormat.hxx

```diff
@@ -11,21 +11,21 @@ namespace portaudio
 {
 
 
-	//////
-	/// @brief PortAudio sample data formats.
-	///
-	/// Small helper enum to wrap the PortAudio defines.
-	//////
-	enum SampleDataFormat
-	{
-		INVALID_FORMAT	= 0,
-		FLOAT32			= paFloat32,
-		INT32			= paInt32,
-		INT24			= paInt24,
-		INT16			= paInt16,
-		INT8			= paInt8,
-		UINT8			= paUInt8
-	};
+    //////
+    /// @brief PortAudio sample data formats.
+    ///
+    /// Small helper enum to wrap the PortAudio defines.
+    //////
+    enum SampleDataFormat
+    {
+        INVALID_FORMAT  = 0,
+        FLOAT32         = paFloat32,
```

Comment:
The expand tool seems to handle tabs in the middle of lines very well.

Reply:
Yeah, it has an option to only convert tabs at the start of lines, but after seeing things like this nicely converted I used it without that.